itests/eth_fee_history_test.go

```diff
@@ -29,95 +46,109 @@ func TestEthFeeHistory(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
-	// Wait for the network to create 20 blocks
+	heads, err := client.ChainNotify(ctx)
+	require.NoError(err)
+
+	// Save the full view of the tipsets to calculate the answer when there are null rounds
+	tsHeights := []int{1}
```

Comment:
Why does this need to be preloaded with 1?

Reply:
It seems that ChainNotify doesn't send the tipset with height = 1

Reply:
Hm, that suggests raciness. The ChainNotify is probably starting on tipset 2. Note that ChainNotify sends a "current" notification on subscribe, and the subscription might be starting on height 2. Do we absolutely need to use ChainNotify here? Alternatively you can walk the chain and populate tsHeights after the fact, i.e. after pausing the miner?

Reply:
ChainNotify being called before the miner starts resolves my raciness concerns. Starting at 2 is probably an artifact from the testkit, and the way we set up miners.